src/Microsoft.AspNetCore.SystemWebAdapters.Authentication/IRemoteAuthenticationMetadata.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+/// <summary>
+/// Metadata for determining whether the remote authentication handler should
+/// be enabled for an endpoint or not. Remote authentication enables useful migration
+/// scenarios but adds unnecessary performance overhead in cases where it isn't needed.
+/// </summary>
+public interface IRemoteAuthenticationMetadata
```

Comment:
Why do we need this? Can't we treat it like any other auth handler?

Reply:
Can we remove this?

Reply:
Yea, I'll pull this out and we can just have options to either make this scheme the default scheme (which would be like enabling the metadata everywhere) or not (in which case, users will have to specify specific endpoints this scheme is used with).

Reply:
Ok, this is removed now. Instead, users specify whether to make remote auth the default or not when registering SystemWebAdapter services for it.